testament/categories.nim

```diff
@@ -527,6 +555,10 @@ proc isJoinableSpec(spec: TSpec): bool =
     spec.outputCheck != ocSubstr and
     spec.ccodeCheck.len == 0 and
     (spec.targets == {} or spec.targets == {targetC})
+  if result:
+    for dir in parentDirs(spec.file, inclusive=false):
```

Comment:
So my personal config in ``~`` disables all of these? I don't think so. Testament should test the tests, it's not required to replicate the compiler's terrible config logic here.

Reply:
That we keep it simple. This part of the logic can also be removed if the tests get patched with ``joinable: false``.

Reply:
kept it like before (it only look in parent dir); but also look for config.nims / myfile.nims ; these are needed as we'll want to test for that